src/Page/Object.elm

```diff
@@ -502,81 +535,125 @@ itemListView db selectedImpact results query =
                     }
 
             Ok items ->
+                let
+                    resultItems =
+                        Simulator.extractItems results
+                in
                 div [ class "table-responsive" ]
                     [ table [ class "table mb-0" ]
                         [ thead []
                             [ tr [ class "fs-7 text-muted" ]
                                 [ th [ class "ps-3", scope "col" ] [ text "Quantité" ]
-                                , th [ scope "col" ] [ text "Procédé" ]
-                                , th [ scope "col" ] [ text "Densité" ]
+                                , th [ scope "col" ] [ text "Composant" ]
                                 , th [ scope "col" ] [ text "Masse" ]
                                 , th [ scope "col" ] [ text "Impact" ]
                                 , th [ scope "col" ] []
                                 ]
                             ]
-                        , Simulator.extractItems results
+                        , resultItems
                             |> List.map2 (itemView selectedImpact) items
+                            |> List.concat
                             |> tbody []
                         ]
                     ]
     , addItemButton db query
     ]
 
 
-itemView : Definition -> ( Query.Amount, Process ) -> Results -> Html Msg
-itemView selectedImpact ( amount, process ) itemResults =
-    tr []
+itemView : Definition -> ( Query.Quantity, String, List ( Query.Amount, Process ) ) -> Results -> List (Html Msg)
+itemView selectedImpact ( quantity, name, processes ) itemResults =
+    [ tr []
         [ td [ class "ps-3 align-middle" ]
             [ div [ class "input-group", style "min-width" "180px" ]
                 [ input
                     [ type_ "number"
                     , class "form-control text-end"
-                    , amount |> Query.amountToFloat |> String.fromFloat |> value
-                    , step <|
-                        case process.unit of
-                            "kg" ->
-                                "0.01"
-
-                            "m3" ->
-                                "0.00001"
-
-                            _ ->
-                                "1"
+                    , quantity |> Query.quantityToInt |> String.fromInt |> value
+                    , step "1"
```

Comment:
Please add a `min` attribute so we can't enter a negative number:

![image](https://github.com/user-attachments/assets/c70f96c1-d518-48eb-8e17-404a5462c916)

Also, how about forbidding entering `0` while we have a delete button? 🤔